assets/css/style.css

```diff
@@ -291,3 +318,64 @@ button:active {
 .about-myself p {
   margin-bottom: 80px;
 }
```

Comment:
- [OPTIONAL] One of the core requirements of this project is to implement responsive design. I can see that you have given the exact width to your `body` tag in style.css. it made your design unresponsive. I will highly recommend you to make your design responsive from 375px to the desktop width which is around `768px`
    - Minimum mobile screen size should be 375px. Errors in screen sizes less than 375px should be ignored. Otherwise, mobile version should be responsive until the desktop screen size.